src/components/admin/sidebar/Sidebar.scss

```
@@ -133,24 +131,22 @@
 		position: relative;
 		align-items: center;
 		justify-content: space-between;
-		margin: 0 10px;
+		margin: 0 var(--s-spacing-xx-small);
 		border-radius: 4px;
```

aqui podemos usar a variável
```suggestion
		border-radius: var(--s-border-radius-small);
```

---

perfeito